docs/logging/helium-logger.html

```diff
@@ -102,31 +102,26 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">HeliumLogger</h1>
-        <p>Helium Logger is a lightweight Swift logging framework which supports logging to standard output.</p>
+        <p><a target="_blank" href="https://github.com/IBM-Swift/HeliumLogger"> Helium Logger</a> is a lightweight Swift logging framework which supports logging to standard output.</p>
+        <p>In this guide we are going to show you how to add Helium Logger to a Kitura server, log messages using the logger and set the level of logging that should be displayed.</p>
         <div class="info">
-          <p>If you created your server using `kitura init` or the macOS app you can skip to step 3.</p>
+          <p>If you don't have a Kitura server set up, follow our <a target="_blank" href="https://www.kitura.io/docs/getting-started/create-server.html"> Create a server</a> guide.</p>
```

Comment:
Why does this sentence have "set up" in the middle and all the others don't. Let's be consistent.